### DISCUSSION TITLE
The meaning of value in attribs* files for data assimilation

### DISCUSSION THREAD

**Discussion by Shuzunyun** [NASA-LIS/LISF#869] [General]
Hi,

Does anybody know the meaning of values in attribs* files? For example, I changed the varmin in state_attribs.txt for assimilation, and the results changed significantly. Could anybody explain how the varmin control the process of assimilation?

**Comment by dmocko** [comment]
Hi,

Sorry for the slow response.  The LIS Users' Guide explains the values.  Specifically for varmin (and varmax) - these are the values which control the boundaries after perturbation.  For example, this setting in the forcing attributes file:

#varmin varmax
Incident Shortwave Radiation Level 001
0.0 1300.0

...will not allow the shortwave radiation to be perturbed below 0.0 or above 1300.0.  These values are to enforce that after perturbation there are still physically realistic values for these forcings.  The same varmin/varmax behavior also applies for state and observations perturbations.

David

**Reply by Shuzunyun** [reply]
Hi,

Thanks for your explanation, that helps a lot.

Shu Zunyun
